### PR TITLE
feature: code of conduct

### PR DESCRIPTION
**Pull request by corinneling**
# Details

## Description

* Adds a code of conduct to the website
* Updates footer styling to prevent 'scroll to top arrow' overlapping links

## Steps to QA

- [ ] Scroll to the bottom of the page
- [ ] Test that all other footer links work
   - [ ] T&C
   - [ ] Privacy 
   - [ ] Cookies
- [ ] Test that the Code of Conduct Modal works

**Note:**
- I used [names] as a placeholder for the names of the Coding Coach team that users can contact if they need to report any form of harassment - **this will need to be updated**
- And I am not sure what we will use to let users report anonymously so I left a placeholder for that as well - **this will need to be updated**

## Issue

#310

### CODE REVIEW

File: src/config/i18n/es/messages.json [hunk 1]
```diff
@@ -17,6 +17,7 @@
   "dashboard.navigation.tab.mentorships": "Mentores",
   "dashboard.navigation.tab.messages": "Mensajes",
   "dashboard.navigation.tab.settings": "Ajustes",
+  "footer.legal.conduct": "",
```
Comment (tureey) [inline src/config/i18n/es/messages.json]: Spanish translation 🇪🇸 : Código de conducta

Comment (corinneling) [reply src/config/i18n/es/messages.json]: Thank you! 😄

Comment (tureey) [reply src/config/i18n/es/messages.json]: Your welcome 😄

File: src/pages/static/CodeOfConduct.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  * Harassing photography or recording, including logging online activity for harassment purposes.
+  * Unwelcome sexual attention such as sexual images or behaviour in spaces where they’re not appropriate.
+  * Pattern of inappropriate social contact, such as requesting/assuming inappropriate levels of intimacy with others
+  * Continued one-on-one communication after requests to cease.
+  * Deliberate “outing” of any aspect of a person’s identity without their consent except as necessary to protect vulnerable people from intentional abuse.
+  * Publication of non-harassing private communication.
+
+Members who violate this code of conduct will be approached by someone of the Coding Coach leadership team and asked to stop immediately. Members may also be banned from the Coding Coach slack, blocked on social media, and removed from the website. 
+
+If someone makes you or anyone else feel unsafe or unwelcome, please report it as soon as possible. To report an incident of harassment you can either directly message one of leadership team, _[names]_, on slack. If you would rather rename anonymous please fill out _[our report form]_.
```
Comment (crysfel) [inline src/pages/static/CodeOfConduct.md]: I think we should use google docs for anonymous reports, maybe later we can build something else but for now that should be more then enough

Comment (emmawedekind) [reply src/pages/static/CodeOfConduct.md]: Love the idea! I'll set it up!

Comment (emmawedekind) [reply src/pages/static/CodeOfConduct.md]: Here is the link to report an incident: https://forms.gle/bcSWqNNcsdo3zDD17

**Comment by emmawedekind** [comment]
I added a link to the Google Form where users can report abuse: https://forms.gle/bcSWqNNcsdo3zDD17

**Comment by corinneling** [comment]
Awesome! thanks 😊 
I've added a link to the form you made @emmawedekind. Let me know if there is anything else

**Comment by emmawedekind** [comment]
If you can please rebase against develop we will be good to go! Thanks so much!

**Pull request merged**